CHANGELOG.md

```diff
@@ -6,11 +6,30 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [1.3.4] 01.11.2024
 
+### Added
```

Comment:
This should go in the unreleased.

Reply:
Done on [`5f44fbf`](https://github.com/polkadot-fellows/runtimes/pull/490/commits/5f44fbfa61701d75e58610f7d4e7031a4e666361)